sdk/appconfiguration/azure-data-appconfiguration/src/main/java/com/azure/data/appconfiguration/ConfigurationAsyncClient.java

```diff
@@ -106,7 +106,7 @@
      *
      * @param setting The setting to add to the configuration service.
      * @return The {@link ConfigurationSetting} that was created, if a key collision occurs or the key is an invalid
-     * value (which will also throw HttpResponseException described below).
+     * value(which will also throw HttpResponseException described below).
```

Comment:
It feels like you did a Ctrl+R to replace all these instances. Before a parenthesis should be a space in these sentences.